black.py

```diff
@@ -1871,24 +1973,24 @@ def visit_factor(self, node: Node) -> Iterator[Line]:
     def __post_init__(self) -> None:
         """You are in a twisty little maze of passages."""
         v = self.visit_stmt
-        Ø: Set[str] = set()
+        empty_set: Set[str] = set()
```

Comment:
Please avoid making changes unrelated to the intended effect of your PR.

Reply:
`test_black` was failing for `pypy3.6` as it does not support unicode symbols. Alternative is to skip this test for `IS_PYPY` flag.